sds_data_manager/lambda_code/SDSCode/dependency_config.csv

```diff
@@ -67,7 +67,8 @@ hit, l1a, hk, hit, l1b, hk, HARD, DOWNSTREAM
 
 # <---- IDEX Dependencies ---->
 
-idex, l0, raw, idex, l1, sci, HARD, DOWNSTREAM
+idex, l0, raw, idex, l1a, sci, HARD, DOWNSTREAM
+idex, l1a, raw, idex, l1b, sci, HARD, DOWNSTREAM
```

Comment:
Is `sci` the name of the file descriptor that IDEX wants for all their L1 products?
```suggestion
idex, l1a, sci, idex, l1b, sci, HARD, DOWNSTREAM
```

Reply:
Oops, good catch!